### DISCUSSION TITLE
Kötelező ügyfélszolgálat elérhetősége

### DISCUSSION THREAD

**Discussion by wlln22** [nav-gov-hu/eRECEIPT#32] [Kérdések és válaszok]
A rendelettervezet szerinti elvárás, hogy a forgalmazó rendelkezzen az üzemeltetők tájékoztatására – különösen a hibabejelentések intézésére, az üzemeltetéssel kapcsolatos kérdések megválaszolására – szolgáló ügyfélszolgálattal, telefonhálózaton keresztül történő vagy online eléréssel. Van-e bármilyen kötöttség ezzel kapcsolatban? Például, hogy 7/24-ben legyen elérhető ügyfélszolgálat? Milyen elérhetőséggel kell rendelkezzen az ügyfélszolgálat?

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
